frontend/src/concepts/areas/utils.ts

```diff
@@ -72,18 +72,18 @@ export const isAreaAvailable = (
     ? Object.values(requiredComponentsState).every((v) => v)
     : true;
 
-  const requiredCapabilitiesState =
-    requiredCapabilities && dsciStatus
-      ? requiredCapabilities.reduce<IsAreaAvailableStatus['requiredCapabilities']>(
-          (acc, capability) => ({
-            ...acc,
-            [capability]: dsciStatus.conditions.some(
-              (c) => c.type === capability && c.status === 'True',
-            ),
-          }),
-          {},
-        )
-      : null;
+  // If DSCI is not available, we shouldn't assume it has met the required capabilities
+  const requiredCapabilitiesState = requiredCapabilities
```

Comment:
Why the change 🤔 We have this for DSC

Reply:
This is part of the issue that i discovered, if DSCI returns a 404 (it's happening in a cluster cause it lacks the permission), it will later return true as if the condition is fulfilled.

Reply:
We shouldn't get that state as dsci should be present every time, but we met this specific edge case @andrewballantyne

Reply:
This is true of everything -- including the DSC -- this is not a good reason to change the logic on the frontend.

Reply:
There is a reason we have the template "test on cluster" checkbox. Because of weird permission things like this.

Reply:
> There is a reason we have the template "test on cluster" checkbox. Because of weird permission things like this.
Yes, and that was on me, I missed that cause when we shipped this the operator side wasn't even completed, so I didn't have the ability to test it on cluster, just mocking my own responses, and that's on me.

On the other hand, I've tested it right now and it seems that we can get the same result with `npm run start:dev:ext` which will give us much more flexibility to test permissions on clusters

Reply:
I'm gonna remove the logic change, keep the tests and keep the permission.